ovssubnet/controller/kube/kube.go

```diff
@@ -72,6 +91,44 @@ func (c *FlowController) manageLocalIpam(ipnet *net.IPNet) error {
 	return nil
 }
 
+func (c *FlowController) SetupIptables(fw *firewalld.Interface, containerNetwork string) error {
+	if fw.IsRunning() {
+		err := fw.EnsureRule(firewalld.IPv4, "nat", "POSTROUTING", 0,
+			[]string{"-s", containerNetwork, "!", "-d", containerNetwork, "-j", "MASQUERADE"})
+		if err != nil {
+			return err
+		}
+		err = fw.EnsureRule(firewalld.IPv4, "filter", "INPUT", 0,
+			[]string{"-p", "udp", "-m", "multiport", "--dports", "4789", "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"})
+		if err != nil {
+			return err
+		}
+		err = fw.EnsureRule(firewalld.IPv4, "filter", "INPUT", 0,
+			[]string{"-i", "tun0", "-m", "comment", "--comment", "traffic from docker for internet", "-j", "ACCEPT"})
```

Comment:
@pravisankar, we assume the ovs bridge is "br0" in both setup.sh and AddOFRules() as well... should we also pass that as an argument?

The arguments to the setup script are getting pretty confusing...